update(UI overview): add bonita apps (#1362)
bonitasoft/bonita-doc: md/bdm-management-in-bonita-bpm-portal.md

# Business Data Management

This page explains what a user with the _Administrator_ profile in Bonita Portal or in the [Bonita Administrator Application](admin-application-overview.md) can see and do about Business Data Model (BDM) and BDM Access Control.

> Processes using Business Objects with persisted data need to run with a Business Data Model deployed in the Bonita Portal.  

Reviewer: ```suggestion
Processes using Business Objects with persisted data need to run with a Business Data Model deployed in Bonita Portal.  
```

On top of that, it is possible to define access rights on some attributes of business objects based on the user profile. This file is also managed in Bonita Portal.

Here is a view of the page:
![BDM in UI](images/UI2021.1/bdm-management.png)<!--{.img-responsive}-->

## Manage the BDM

::: warning
Only one model can be deployed at a time in Bonita Portal, so make sure it contains the Business Object definitions used in all processes that will run within this tenant.

> The Business Objects must match the structure used by the deployed processes. Make sure that when modifying the Business Data Model, the process is modified accordingly.

Reviewer: ```suggestion
The Business Objects must match the structure used by the deployed processes. Make sure that when modifying the Business Data Model, the processes are modified accordingly.
```

This limitation is well known and will be addressed in a future Bonita version.
:::

<a id="installAccessControl"/>
## Manage BDM Access Control

::: info
**Note:** For Enterprise, Performance, and Efficiency editions only.
:::

It is possible to define BDM Access Control rules in Bonita Studio and import them in Bonita Portal.  
The Access Control rules must match the Business Data model.  

::: info
**:fa-info-circle: Note:** Contrary to the BDM definition, the deployment of a BDM Access Control file does not require pausing the BPM services during the operation. 
:::
1. [Export the BDM Access Control file](bdm-access-control.md) from Bonita Studio where it was designed.

> 1. In Bonita Portal, log in as a user with the _Administrator_ profile, or as the Technical user (default login 'install', default password 'install') 

Reviewer: I would remove the default username/password as it is necessarily changed by Sam

Author: ok
